shell2http.go

```diff
@@ -291,7 +294,7 @@ func getShellHandler(appConfig Config, path string, shell string, params []strin
 			shellOut, err = osExecCommand.Output()
 		}
 
-		if err != nil {
+		if err != nil && !appConfig.showErrors {
```

Comment:
If `showErrors` is set, is it recognizable now that the command actually failed? The `exec error` line was useful for that - would it make sense to keep that?